app/main.py

```diff
@@ -1,8 +1,35 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    people_dict = {}
+
+    for i in people:
+        name = i["name"]
+        age = i["age"]
+        person = Person(name, age)
+        people_dict[name] = person
+
+    for i in people:
```

Comment:
Choose a more descriptive variable name, and avoid using single-letter variable names.